screed/__main__.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2016, The Regents of the University of California.
+
+from __future__ import absolute_import, print_function
+
+import argparse
+import sys
+
+from . import createscreed
+from . import dump_fasta
+from . import dump_fastq
+
+
+class ScreedCommands(object):
+
+    def __init__(self):
+        parser = argparse.ArgumentParser(
+            description="",
+            usage='''screed <command> [<args>]
+
+Available:
+
+    db <filename>               Creates a screed database.
+    dump_fasta <db> <output>    Convert a screed database to a FASTA file
+    dump_fastq <db> <output>    Convert a screed database to a FASTQ file
```

Comment:
I still think `fastq` and `fasta` are simpler and more concise, but for now this is fine.